libs/SalesforceSDK/src/com/salesforce/androidsdk/auth/interfaces/NativeLoginManager.kt

```diff
@@ -72,6 +82,70 @@ interface NativeLoginManager {
      */
     fun getFallbackWebAuthenticationIntent(): Intent
 
+    // region Salesforce Identity API Headless Registration Flow
+
+    /**
+     * Submits a request to start a user registration to the Salesforce Identity
+     * API headless registration flow.  This fulfills step four of the headless
+     * registration flow.
```

Comment:
You probably want to link the documentation this is referencing (like you did with password reset) for this to make sense to the reader.   Same with `completeRegistration`.

Reply:
🦅 I award @brandonpage the "Eagle Eyes" award for noticing that!